scenes/strategic-point-menu.gd

```diff
@@ -15,6 +15,10 @@ func _on_Slot_pressed(is_slot_active, slot: Slot):
 		buy_tower(slot)
 
 
+func get_slots():
+	return $RadialContainer.get_children()
```

Comment:
Can you add `$RadialContainer` as a onready var?

Reply:
yeppa